app/helpers/components_helper.rb

```diff
@@ -11,10 +11,6 @@ def offence_dates(offence)
     render OffenceDatesComponent.new(offence:)
   end
 
-  def decision_results(results)
-    render DecisionResultComponent.with_collection(results)
-  end
-
```

Comment:
☝️ this was used on the completes page to show multiple results. Not needed now that page has been removed.